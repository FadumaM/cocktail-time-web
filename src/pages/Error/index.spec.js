import React from "react";
import { render } from "@testing-library/react";
import ErrorPage from "./index";

describe("Error", () => {
  it("should render correctly", () => {
    const { baseElement } = render(<ErrorPage />);
    expect(baseElement).toMatchSnapshot();
  });
});
