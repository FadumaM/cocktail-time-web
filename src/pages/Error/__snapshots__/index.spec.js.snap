// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<body>
  <div>
    <div
      class="error-page"
    >
      <h1>
        Sorry error has occured
      </h1>
      <a
        href="/"
      >
        Go Home 
      </a>
    </div>
  </div>
</body>
`;
